qa/L0_backend_python/examples/test.sh

```diff
@@ -440,4 +442,7 @@ else
     echo -e "\n***\n*** Example verification test FAILED.\n***"
 fi
 
+pip3 uninstall -y numpy
+pip3 install "numpy<2"
```

Comment:
In L0_backend_python, the way `examples/test.sh` is called is along the lines of:
```
    setup_virtualenv

    set +e
    (cd ${TEST} && bash -ex test.sh)

    if [ $? -ne 0 ]; then
        echo "Subtest ${TEST} FAILED"
        RET=1
    fi
    set -e

    deactivate_virtualenv
```
So re-installing a lower version of numpy is unnecessary here because the virtualenv created for the examples subtest is thrown away and a new one is created for future next subtest.